Source/WebCore/platform/network/ResourceRequestBase.h

```diff
@@ -114,6 +114,7 @@ class ResourceRequestBase {
         bool m_didFilterLinkDecoration : 1 { false };
         bool m_isPrivateTokenUsageByThirdPartyAllowed : 1 { false };
         bool m_wasSchemeOptimisticallyUpgraded : 1 { false };
+        bool m_hasCrossOriginRedirect : 1 { false };
```

Comment:
This increases sizeof(ResourceRequest).  Can it share a bit with something else, like ResourceLoadPriority?

Also, can NetworkLoadMetrics.hasCrossOriginRedirect be used instead?  We already gather this information and pipe it into WebCore.  Having a cross origin redirect is not conceptually part of a request.